math256/src/result.rs

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use crate::errors::Error;
+use ethnum::u256;
+use fluence::marine;
+use num_traits::ToPrimitive;
+
+#[marine]
+#[derive(Default, Debug)]
+pub struct MathResult {
+    /// u256 string representation
+    pub value: String,
+    // 0 means success
+    pub ret_code: u8,
+    /// contains error as a string
+    pub err_msg: String,
+}
+
+impl From<std::result::Result<u256, Error>> for MathResult {
```

Comment:
Is `std::result::` necessary here?

Reply:
sure

Reply:
But why? You don't define your own type `Result` anywhere. And using `impl From<std::result::Result<u256, Error>> for MathResult` is fine here, because it's already known that `Result` is from `std::result`. F.e. on the same way you use `pub fn add(lhs: String, rhs: String) -> Result<u256, Error>` with just `Result` without full path.

Reply:
sorry, i meant deleted prefix.